src/OpenTelemetry.Extensions.Hosting/CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Released 2023-Feb-10
   traces and/or metrics.
   ([#4174](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4174))
 
+* Removed deprecated extensions: `AddOpenTelemetryTracing`,
```

Comment:
Unreleased section?

Reply:
Good catch! Not sure what happened there, probably a merge thing.